tests/bindings-test/RuntimeTest.cs

```diff
@@ -57,19 +57,36 @@ public void EvilDeallocatorTest ()
 		}
 
 		[Test]
-		public void MainThreadDeallocationTest ()
+		public void MainThreadDeallocationTestPriority ()
 		{
 #if OPTIMIZEALL
 			if (!TestRuntime.IsLinkAll)
 				Assert.Ignore ("This test must be processed by the linker if all optimizations are turned on.");
 #endif
 
-			ObjCBlockTester.CallAssertMainThreadBlockRelease ((callback) => {
```

Comment:
Also keep the existing test code (it should continue to work), while just adding new tests instead.